README.md

```diff
@@ -1,128 +1,102 @@
 # ethereum-cryptography
 
-[![npm version][1]][2] [![license][3]][4]
+[Audited](#security) pure JS library containing all Ethereum-related cryptographic primitives. Implemented with 6 [noble & scure](https://paulmillr.com/noble/) dependencies.
 
-[Audited](#security) pure JS library containing all Ethereum-related cryptographic primitives.
-
-Included algorithms, implemented with just 5 [noble & scure](https://paulmillr.com/noble/) dependencies:
-
-* [Hashes: SHA256, keccak-256, RIPEMD160, BLAKE2b](#hashes-sha256-keccak-256-ripemd160-blake2b)
-* [KDFs: PBKDF2, Scrypt](#kdfs-pbkdf2-scrypt)
-* [CSPRNG (Cryptographically Secure Pseudorandom Number Generator)](#csprng-cryptographically-strong-pseudorandom-number-generator)
-* [secp256k1 elliptic curve](#secp256k1-curve)
-* [BIP32 HD Keygen](#bip32-hd-keygen)
-* [BIP39 Mnemonic phrases](#bip39-mnemonic-seed-phrase)
-* [AES Encryption](#aes-encryption)
-
-**April 2023 update:** v2.0 is out, switching
-[noble-secp256k1](https://github.com/paulmillr/noble-secp256k1) to
-[noble-curves](https://github.com/paulmillr/noble-curves),
-which changes re-exported api of `secp256k1` submodule.
-There have been no other changes.
-
-**January 2022 update:** v1.0 has been released. We've rewritten the library from
-scratch and [audited](#security) it. It became **6x smaller:** ~5,000 lines of
-code instead of ~24,000 (with all deps); 650KB instead of 10.2MB.
-5 dependencies by 1 author are now used, instead of 38 by 5 authors.
-
-Check out [Upgrading](#upgrading) section and an article about the library:
+Check out [Changelog / Upgrading](#upgrading) and an article about the library:
 [A safer, smaller, and faster Ethereum cryptography stack](https://medium.com/nomic-labs-blog/a-safer-smaller-and-faster-ethereum-cryptography-stack-5eeb47f62d79).
 
 ## Usage
 
-Use NPM / Yarn in node.js / browser:
+> npm install ethereum-cryptography
 
-```bash
-# NPM
-npm install ethereum-cryptography
-
-# Yarn
-yarn add ethereum-cryptography
-```
-
-See [browser usage](#browser-usage) for information on using the package with major Javascript bundlers. It is
-tested with **Webpack, Rollup, Parcel and Browserify**.
+We support all major platforms and runtimes.
```

Comment:
Nit: Would be nice to list what's explicitly supported

Reply:
Yeah, also would be good to somehow test in bun, deno